primitives/primproc/batchprimitiveprocessor.cpp

```diff
@@ -599,8 +601,8 @@ void BatchPrimitiveProcessor::resetBPP(ByteStream& bs, const SP_UM_MUTEX& w, con
 
   /* init vars not part of the BS */
   currentBlockOffset = 0;
-  memset(relLBID.get(), 0, sizeof(uint64_t) * (projectCount + 1));
-  memset(asyncLoaded.get(), 0, sizeof(bool) * (projectCount + 1));
+  memset(relLBID.get(), 0, sizeof(uint64_t) * (projectCount + 2));
```

Comment:
What if we set this constant somewhere to be available globally b/c there are some places where you add this value in EM code also?

Reply:
I can't recall if I added/incremented any constant in the EM code for the AUX column. In the EM code, in addition to passing the lbid for the scan column, we now also pass the lbid for the AUX column.